target/scripts/build/packages.sh

```diff
@@ -97,6 +97,7 @@ function _install_dovecot() {
     dovecot-core dovecot-imapd
     dovecot-ldap dovecot-lmtpd dovecot-managesieved
     dovecot-pop3d dovecot-sieve dovecot-solr
+    dovecot-lua
```

Comment:
Can you add a comment above `DOVECOT_PACKAGES=` to note that `dovecot-lua` is a community contributed package?

It helps to keep this distinction for maintenance. We'll need to do that for some prior community contributed packages too (_you don't need to worry about that though_).

Reply:
A comment was added. Furthermore, a distinction is now made between x86_64 (use Dovecot community repository) and other architectures (use Debian's repository). A comment was added to clarify this.